### PR TITLE
Akif js2 w3

### PR DESCRIPTION
**Pull request by Akif2709**

### CODE REVIEW

File: Week3/homework/step2-3.js [hunk 1]
```diff
@@ -4,10 +4,13 @@
...
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
+  if (num > 0) {
+    for (let i = 0; i < num; i++) {
+      result += str;
+    }
+  } else {
```
Comment (paulbremer) [inline Week3/homework/step2-3.js]: You are not really returning anything useful so you could just skip the `else` here.

File: Week3/homework/step2-7.js [hunk 2]
```diff
@@ -21,3 +21,11 @@ f2(y);
 console.log(y);
 
 // Add your explanation as a comment here
+/* Primitive variables can not be changed by function, but array or object can be changed. Because when we write array or object
+as a argument, it is approached with its reference/address. And when we change object in  function, we are changing 
+the referenced/original object. Dont ask why, it is Javascript : )
```
Comment (paulbremer) [inline Week3/homework/step2-7.js]: But we are asking why! I think you get the general idea, try to explain a bit more by using the code from the task.

Comment (Akif2709) [reply Week3/homework/step2-7.js]: I added more explanation using codes as much as possible. Thanks for review Paul

Comment (paulbremer) [reply Week3/homework/step2-7.js]: 👍

**Pull request closed without merging**